cluster/v1alpha1/types_rolloutstrategy.go

```diff
@@ -0,0 +1,88 @@
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
this is not a runtime object so line 7 is not needed. see here https://pkg.go.dev/k8s.io/gengo/examples/deepcopy-gen